GOVERNANCE.md

```diff
@@ -166,19 +189,11 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
-
-The nomination passes if no collaborators oppose it after one week. In the case
+The nomination passes if no collaborators oppose it after one week, and if the
+nominee publicly accepts it. In the case
```

Comment:
Why do we explicitly need to define "publicly accepts it"? What does this step solve/fix? What's the purpose?

Reply:
There is also cases where we don't ask it publicly: https://github.com/nodejs/node/issues/55053.

I think explicitly adding this doesn't make a difference, and if it doesn't make a difference what's the goal of adding it?